server/game/cards/13.5-TB/Yoren.js

```diff
@@ -7,7 +7,7 @@ class Yoren extends DrawCard {
                 onCardEntersPlay: event => event.card === this
             },
             target: {
-                cardCondition: card => card.getType() === 'character' && card.owner !== this.controller && card.getPrintedCost() <= 5 && card.location === 'play area'
+                cardCondition: card => card.getType() === 'character' && card.getPrintedCost() <= 5 && card.location === 'play area'
```

Comment:
I thought about this before submitting and thought that the succesful establishment of a lasting effect is affecting the game state. To be honest the rules are pretty obscure in this regard: "A card ability can only be initiated if its effect has the potential to change the game state. This potential is assessed without taking into account the consequences of the cost payment or any other ability interactions." Whatever those other interactions are...

If you are sure about the rule, I will implement that behaviour though.

Reply:
Just seen the answer on discord. Seems ready to merge then, right?

Reply:
Yup! Sorry, I answered mplain on there then went to lunch. 😄 So this is both rules accurate and has real-world use cases (refresh control on a character you have another take control lasting effect on to prevent it from expiring).